cap-ci/cap-nightly.yaml

```diff
@@ -0,0 +1,65 @@
+workertags:
```

Comment:
How about naming this file `cap-nightly-sa.yaml` since you are not setting HA?

Reply:
Maybe I'm just confused by the `availabilities` settings...

Reply:
yeah, I think you got confused. This file handles SA, HA, diego, eirini etc.